01 - Community/Obsidian Roundup/🌠 Better Android performance & custom sorting of files.md

```diff
@@ -0,0 +1,104 @@
+---
+link: https://www.obsidianroundup.org/2022-02-05/
+author: Eleanor Konik
+published: 2022-02-05T13:30:00
+publish: true
+---
+
+# 🌠 Better Android performance & custom sorting of files
```

Comment:
I think it's really helpful to see the publication date at the start of the file name.

It's a pain that the date convention changed at some point, but I feel it's better to have a date at the start than not.

For example, the only reason I was able to identify the last issue added was because of the file name.

It also makes it easy to spot gaps in the series.

Also, when looking at (future) backlinks, it helps to be able to understand how old the news is that one is looking at.

Reply:
> I think it's really helpful to see the publication date at the start of the file name.
> 
> It's a pain that the date convention changed at some point, but I feel it's better to have a date at the start than not.
> 
> For example, the only reason I was able to identify the last issue added was because of the file name.
> 
> It also makes it easy to spot gaps in the series.
> 
> Also, when looking at (future) backlinks, it helps to be able to understand how old the news is that one is looking at.

I see the disappearance of dates from the names of Roundup issues is because @eleanorkonik changed the naming convention - I'm seeing the difference in my emails.

Reply:
Thank you, I was holding off changing the file names unless someone told me to. Will do that! :sweat_smile: I would suggest putting the usual YY-MM-DD in front of all files but the ones with YY.MM.DD – would you agree or do you have a better idea?

Reply:
Ah, and one more thing: Should I strip the emoji or leave it? :thinking:

Reply:
> Thank you, I was holding off changing the file names unless someone told me to. Will do that! 😅 I would suggest putting the usual YY-MM-DD in front of all files but the ones with YY.MM.DD – would you agree or do you have a better idea?

I would have suggested putting YYYY-MM-DD in front of the ones that don't already have a date on it. That may be what you meant... It's the newer pattern anyway, so matches the existing files from 2022.



> 
> It's a bit awkward with the earliest editions.

Reply:
> Ah, and one more thing: Should I strip the emoji or leave it? 🤔

I would remove it - Personally I don't think it adds any information and is a distraction from the pattern of naming...

(It's fine in emails - that's a different context...)

Reply:
> It's a bit awkward with the earliest editions.

It might be worth a separate discussion on Discord about whether to rename the initial ones to use a hyphen instead of a dot... I'd be happy to do that work.

Reply:
Added dates to file names which were missing one in YYYY-MM-DD format (you were right, I meant to write that) and removed the emojis... Now I just don't know what to do with the rest. :thinking: